pkgs/by-name/fa/factorio_achievement_enabler_linux/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "Factorio Achievement Enabler for Linux";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "UnlegitSenpaii";
+    repo = "FAE_Linux";
+    rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
This is a new shorthand for the refs/tags/ specification.